sdk/src/cache/cache_client.rs

```diff
@@ -343,6 +391,54 @@ impl CacheClient {
         request.send(self).await
     }
 
+    /// Gets a batch of items from a Momento Cache
+    ///
+    /// # Arguments
+    ///
+    /// * `cache_name` - name of cache
+    /// * `keys` - list of keys to fetch
+    ///
+    /// # Examples
+    /// Assumes that a CacheClient named `cache_client` has been created and is available.
+    /// ```
+    /// # fn main() -> anyhow::Result<()> {
+    /// # use momento_test_util::create_doctest_cache_client;
+    /// # tokio_test::block_on(async {
+    /// # let (cache_client, cache_name) = create_doctest_cache_client();
+    /// use std::convert::TryInto;
+    /// use std::collections::HashMap;
+    /// use momento::cache::GetResponse;
+    /// # cache_client.set(&cache_name, "key1", "value1").await?;
+    /// # cache_client.set(&cache_name, "key2", "value2").await?;
+    ///
+    /// let results_map: HashMap<String, GetResponse> = cache_client.get_batch(&cache_name, vec!["key1", "key2"]).await?.into();
```

Comment:
the `get_batch` examples get results as `HashMap<String, GetResponse>` but I can change them to another type if there's another one that may be more commonly accessed